docs/general/performance.md

```diff
@@ -63,6 +62,28 @@ new Chart(ctx, {
 });
 ```
 
+### Automatic data decimation during draw
```

Comment:
It might be worth mentioning in the data decimation section above that it can be done automatically for line charts

Reply:
Its still better to do it beforehand though. But I can add a note.